src/styles/main.css

```diff
@@ -0,0 +1,45 @@
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.tasks {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  color: blue;
+  background: red;
+}
+
+.head {
+  color: red;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header {
+  margin: 0 10px;
+}
+
+.navside {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  margin: 0 10px;
+}
+
+.home:hover {
+  color: blue;
+}
+
+.contact:hover {
+  color: blue;
```

Comment:
- [ ] KIndy your to-do list should be a clone of the part of the minimalist project captured in the video below a the requirements says

https://www.youtube.com/watch?v=AcUd-_Yjjqg